CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/scatter/Scatter3dArranger.java

```diff
@@ -179,7 +179,7 @@ public void arrange(final GraphWriteMethods wg) throws InterruptedException {
     private float scaleValue(float value, boolean logarithmic) {
         if (logarithmic) {
             if (value <= 0.0) {
-                return 0.0f;
+                return (float) Math.log10(-value) * -1;
```

Comment:
You could do this in one operation using `Math.log10(Math.abs(value)) * Math.signum(value)`.

Reply:
Hmm, I actually broke it with the request :( 
log10(0) = -Infinity is the thing I want to avoid.

Reply:
Seems that log10(0) * 0 still gives NaN so kept the 0.0 case in.

Reply:
Good catch.